apps/src/templates/lessonOverview/activities/LevelDetailsDialog.jsx

```diff
@@ -205,6 +207,25 @@ class LevelDetailsDialog extends Component {
     }
   }
 
+  recordSeeFullLevelClick = (level, scriptLevel) => {
+    firehoseClient.putRecord(
+      {
+        study: 'lesson-plan',
+        study_group: 'teacher-lesson-plan',
+        event: 'click-see-full-level',
+        data_json: JSON.stringify({
+          scriptLevelId: scriptLevel.id
+        })
+      },
+      {
+        includeUserId: true,
+        callback: () => {
+          windowOpen(level.url || scriptLevel.url);
```

Comment:
We should avoid using `window.open` whenever possible, for the same reasons that we should avoid using links with `target=_blank`; both are security issues. Prefer `window.location.href`.

Also: we should get rid of this utility. We don't need to define a wrapper method to stub anything for unit tests, and we shouldn't define wrappers that don't actually add anything.

Reply:
It looks like `window.open` [supports](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#noopener) `noopener` and `noreferrer`. It might be worth keeping this util function and ensuring that those are set properly

Reply:
It looks like window.location.href would change the behavior of this feature which is to open a new tab. @Hamms what is the best way to open a new tab considering the security concerns? What about trying Bethany's suggestion? Would it be alright if we did that in a follow up PR in case it impacts other areas?